build/configs/rtl8721csm/board_metadata.txt

```diff
@@ -25,6 +25,8 @@ BL1="km0_boot_all"
 BL2="km4_boot_all"
 KERNEL="km0_km4_image2"
 OTA="km0_km4_image2"
+MICOM="micom"
```

Comment:
I think we don't need to add these into the board metadata.
Because they are not board specific name.
Even the name of `BOOTPARAM` also.
How do you think?